R/meta_regression/treatment_comparison_page.R

```diff
@@ -97,7 +97,9 @@ treatment_comparisons_page_baseline_risk_server <- function(
     output$downloadbaye_comparison <- downloadHandler(
       filename = 'baseline_risk_comparison.csv',
       content = function(file) {
-        write.csv(baye_comp(model(), outcome_measure()), file)
+        write.csv(
+          BaselineRiskRelativeEffectsTable(bnma::relative.effects.table(model(), summary_stat = "ci")), file
```

Comment:
Can you explain this change please? It looks like the function `BaselineRiskRelativeEffectsTable()` is now going to be used for both covariate analysis and baseline risk analysis.

Reply:
You can see why I made the mistake in the first place. There are two server functions!